src/app/components/footer/Footer.ts

```diff
@@ -0,0 +1,34 @@
+import styles from './Footer.module.scss';
+
+export class Footer {
+    private __element: HTMLDivElement;
+
+    constructor() {
+        this.__element = document.createElement('div');
+        this.__element.classList.add(styles.wrapper);
+
+        const footerImageLeft = document.createElement('div');
+
+        const footerTextWrapper = document.createElement('div');
+        const codersCampText = document.createElement('p');
+        codersCampText.innerHTML = '.Coders';
+        const campText = document.createElement('span');
+        campText.innerHTML = 'Camp';
+        codersCampText.appendChild(campText);

```

Comment:
Można to zapisać w jednej linii, przy pomocy `innerHTML`, zamiast tworzyć nowego spana.
```ts
codersCampText.innerHTML = '.Coders<span>Camp</span>'
```